src/conversions/chrono.rs

```diff
@@ -14,14 +14,32 @@
 //! chrono = "0.4"
 // workaround for `extended_key_value_attributes`: https://github.com/rust-lang/rust/issues/82768#issuecomment-803935643
 #![cfg_attr(docsrs, cfg_attr(docsrs, doc = concat!("pyo3 = { version = \"", env!("CARGO_PKG_VERSION"),  "\", features = [\"chrono\"] }")))]
-#![cfg_attr(
-    not(docsrs),
-    doc = "pyo3 = { version = \"*\", features = [\"chrono\"] }"
-)]
+#![cfg_attr(not(docsrs), doc = "pyo3 = { version = \"*\", features = [\"chrono\"] }")]
 //! ```
 //!
 //! Note that you must use compatible versions of chrono and PyO3.
 //! The required chrono version may vary based on the version of PyO3.
+//!
+//! # Example: Convert a PyDateTime to chrono's DateTime<Utc>
+//!
+//! ```rust
+//! use chrono::{Utc, DateTime};
+//! use pyo3::{Python, ToPyObject, types::PyDateTime};
+//!
+//! fn main() {
+//!     pyo3::prepare_freethreaded_python();
+//!     Python::with_gil(|py| {
+//!         // Create an UTC datetime in python
+//!         let py_tz = Utc.to_object(py);
+//!         let py_tz = py_tz.cast_as(py).unwrap();
+//!         let pydatetime = PyDateTime::new(py, 2022, 1, 1, 12, 0, 0, 0, Some(py_tz)).unwrap();
+//!         println!("PyDateTime:\t{pydatetime}");
+//!         // Now convert it to chrono's DateTime<Utc>
+//!         let chrono_datetime: DateTime<Utc> = pydatetime.extract().unwrap();
+//!         println!("DateTime<Utc>:\t{chrono_datetime}");
+//!     });
```

Comment:
I wish there is an interesting example using `inline_python!`. https://docs.rs/inline-python/latest/inline_python/

I think it also helps demonstrate this nicely, although not sure if we want to run doctest since it takes in a new dependency.

Reply:
Didn't know about this project. This would make the examples much more readable, but I'm not sure we want to hide the details of how to do things with PyO3 in the documentation, new users of the library would be confused by the requirement of an external dependency to run PyO3 code. Anyway I think this is out of scope for this PR.

Reply:
Yes, but it's for the users that are coming from inline_python, not pyo3. It was the motivation for me to contribute the initial pull request to chrono, because I wanted to use it in inline_python, and inline_python uses pyo3, I wouldn't have use pyo3 directly.

I think would be good to keep as a separate example given that users coming from inline_python probably have no idea what the pyo3 example is showing.